lib/miq_automation_engine/service_models/miq_ae_service_methods.rb

```diff
@@ -65,8 +65,7 @@ def self.vm_templates
 
     def self.category_exists?(category)
       ar_method do
-        cat = Classification.find_by_name(category)
-        cat.nil? ? false : true
+        Classification.find_by_name(category).present?
```

Comment:
It is; but `Classification#find_by_name` [is (sadly) an actual method](https://github.com/manageiq/manageiq/blob/411fb8278e8e597da7a18cb79349284319ff35b0/app/models/classification.rb#L330-L333), not the dynamic AR one.